hermes/src/network/rpc/types.rs

```diff
@@ -0,0 +1,74 @@
+use {
```

Comment:
This file is mostly the rename of the deleted rpc_price_feed.rs part and moved some other types from rest.rs here.